list.yaml

```diff
@@ -642,6 +642,10 @@
   license: "MIT"
   website: ""
 
+- name: "Franklin"
+  github: "bryanbraun/franklin"
+  license: "MIT"
+  website: ""
```

Comment:
Can you remove the `website:` line? Not needed if there's no website associated.